Dockerfile.dist

```diff
@@ -0,0 +1,42 @@
+# This dockerfile builds a base image that can run a binary version of S2E.
+# First, build S2E using the Dockerfile, then copy /opt/s2e from the resulting image
+# to this image.
+
+FROM ubuntu:16.04
+MAINTAINER Volodymyr Kuznetsov <vova@cyberhaven.io>
+
+# i386 packages are required by IDAPro
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ precise-pgdg main 9.4" \
+        > /etc/apt/sources.list.d/pgdg.list && \
```

Comment:
Also can we aline the trailing `\` at the end of each line? It makes it easier to read.